web/frontend/src/app.vue

```diff
@@ -840,15 +820,69 @@ onMounted(async () => {
     />
 
     <!-- ******* CAMERAS *******  -->
-    <CameraView
+    <v-collapse
```

Comment:
Kinda feels like we still may want to make this a component like `<CamerasList />` or something just for cleanliness and so we can export it as a part of the RC package.

Reply:
Okay sure